src/react-components/preferences-screen.js

```diff
@@ -389,6 +389,10 @@ const preferenceLabels = defineMessages({
     id: "preferences-screen.preference.disable-auto-gain-control",
     defaultMessage: "Disable microphone automatic gain control"
   },
+  audioNormalization: {
+    id: "preferences-screen.preference.audio-normalization",
+    defaultMessage: "Dynamic audio normalization factor (0 is disabled, 4 is recommend)"
```

Comment:
Let me know if you have an idea to simplify the message.

![image](https://user-images.githubusercontent.com/7637832/113464685-87321e00-93e3-11eb-9d21-25d1c1a1eccc.png)

Reply:
I agree that people will not know what this means. Adding additional help information (like you suggested in Discord) is a good idea. 

Maybe this should be exposed as a boolean property - "ON" means "4" and "OFF" means "0". I know that 4 is not perfect for all scenarios, but if it is _usually_ better than 0, then maybe we want to set it to 4 by default and later give people the option to either disable it or set a custom factor.

Reply:
Yeah, I have been thinking of boolean, too, because it's clear to users. But also I still want factor configuration for the test. So what do you think of add normalization configuration to preference as boolean and also keep it in chat command which accept factor 0 to x?

Reply:
It's up to you. We tested the audio normalization factors a couple months ago and I could not tell the difference. How do you intend to figure out what the best default value is (surveys, analytics, user studies)? If you just want this code to be used and let people on the team tell you over time what their preferred number is, maybe just start with what you have (a slider) and change it to a boolean if you ask in 6 months and everyone likes the same value.

Reply:
Dynamic audio normalization attempts to make master volume into a certain range specified with audio normalization factor. The biggest concern about boolean (fix factor number) is the appropriate factor should very depend on users' environment. For example there could be a chance that 4 could be too quiet for speaker users but too loud for ear pad users. So I'm leaning toward a slider first and then change it to boolean if we will get surveys or analytics and realize most of users like the same factor...

The factor number may not be straight forward to users, I would add a help icon which pops dialog explaining what it is.

Reply:
Sorry for changing my opinion again but I changed the Automatic Output Gain Control from a slider to a checkbox because it will be much straightforward to users. I renamed Audio Normalization to Audio Output Gain Control and I speculate users can understand what it is from the name. I started to think that making the preference layout complex by adding an additional dialog just for explaining the factor would be bad.